ssp-balancer/scripts/tx.json

```diff
@@ -0,0 +1,13 @@
+{
+    "96": {
```

Comment:
No need to persist this in a file imo, we can just post as a comment on the relevant PR. Wdyt?